### PR TITLE
Remove pre-commit linter from Web UI

### PR DESCRIPTION
**Pull request by tieujason330**
Removing it because web ui changes aren't frequent enough to warrant husky attempting to run for every commit. Instead, the linter is run if building the UI

### CODE REVIEW

**Comment by ZacBlanco** [comment]
Changes look good. Just want to confirm that what your changes now require eslint to run successfully in order to build the webUI instead of running the linting as a post-build step?

File: webui/common/package.json [hunk 1]
```diff
@@ -27,17 +27,13 @@
     "start-js": "BROWSER=none react-scripts start --watch --recursive",
     "build-js": "react-scripts build --production",
     "start": "npm-run-all --parallel watch-css start-js",
-    "build": "NODE_ENV=production && npm-run-all --parallel build-css build-js",
+    "build": "eslint 'src/**/*.tsx' && NODE_ENV=production && npm-run-all --parallel build-css build-js",
```
Comment (ZacBlanco) [inline webui/common/package.json:30]: Is there a reason to set 
```
NODE_ENV=production && npm-run-all ...`
```
vs 
```
NODE_ENV=production npm-run-all ...
```

I think the ampersand is unnecessary?

Comment (tieujason330) [reply webui/common/package.json]: updated

**Comment by ZacBlanco** [comment]
alluxio-bot, merge this please

**Pull request merged**